wagtail_localize/views/convert.py

```diff
@@ -70,6 +70,15 @@ def convert_to_alias(request, page_id):
                 },
             )
 
+            # Now clean up the old translation
```

Comment:
Probably worth noting here why we delete instead of disable?

Reply:
good shout. done